src/lib/utils/platform.js

```diff
@@ -1,12 +1,30 @@
 import { Platform } from 'react-native'
-import { isMobile as isMobileWeb, isMobileOnly as isMobileWebOnly } from 'mobile-device-detect'
+import * as mobileWebDetect from 'mobile-device-detect'
+import DeviceInfo from 'react-native-device-info'
 import isWebApp from './isWebApp'
 
-export const isMobileReactNative = Platform.OS === 'android' || Platform.OS === 'ios'
+export * from 'mobile-device-detect'
 
-export const isMobile = isMobileReactNative || isMobileWeb
+export const isMobileWeb = mobileWebDetect.isMobile
 
-export const isInstalledApp = isMobileReactNative || isWebApp
+export const isIOSWeb = mobileWebDetect.isIOS
 
-// TODO: RN - Need to implement isMobileOnlyReactNative
-export const isMobileOnly = isMobileReactNative || isMobileWebOnly
+export const isAndroidWeb = mobileWebDetect.isAndroid
+
+export const isIOSNative = Platform.OS === 'ios'
+
+export const isAndroidNative = Platform.OS === 'android'
+
+export const isMobileNative = isIOSNative || isAndroidNative
+
+export const isMobileOnlyNative = !DeviceInfo.isTablet() && isMobileNative
```

Comment:
I didnt see this being used anywhere and its the only usage of react-native-device-info
so maybe we dont need it

Reply:
It is, on line 26 `export const isMobileOnly = isMobileOnlyNative || mobileWebDetect.isMobileOnly` Wrapper.js uses isMobileOnly.

Also, if you are worried about adding react-native-device-info only for this, it was already an indirect dependency, another module we use (I don't recall which one uses it)

Reply:
where was the !isTablet logic used before?
in anycase dont import *

Reply:
src/components/common/layout/Wrapper.js uses `isMobileOnly`. In web isMobileOnly will return false if it's running in a tablet. That doesn't work while running a mobile app, so I'm using isTablet to leave out tablets in that case

Reply:
src/components/dashboard/FeedModalList.js
src/components/common/modal/ModalWrapper.js
src/lib/utils/Orientation.js

Those use isMobileOnly too